hr_course/models/hr_course.py

```diff
@@ -180,6 +183,44 @@ def cancel_course(self):
         for record in self:
             record.write(record._cancel_course_values())
 
+    @api.model
+    def send_email(self, partner):
+        email = partner.email
+        if email and email.strip():
+            email_template = self.env.ref(
+                "hr_course.mail_template_validity_reminder_en"
+            )
+            options = {
+                "email_to": email,
+                "email_from": self.env["res.company"].search([]).email.strip(),
+            }
+            if email_template:
+                email_template.send_mail(self.id, email_values=options, force_send=True)
+                return True
+        raise UserError(
+            _(
+                """Could not send mail to partner %s because it does not\
+                have any email address defined""",
+                partner.display_name,
+            )
+        )
```

Comment:
```suggestion
```
as here you will have a list of users and not one it is better to not raise, if one partner missing email it is better to send all emails, the corrupted address will stay in failed emails